front-end/src/components/CheckoutForm.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { CardElement, injectStripe } from 'react-stripe-elements';
+
+class CheckoutForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      complete: false
+    };
+    this.submit = this.submit.bind(this);
+  }
+
+  //tokenizes the card information by invoking createToken on the stripe prop
+  async submit(ev) {
```

Comment:
Let's change this to far arrow function so we don't have to bind in the constructor
```suggestion
  submit = async (ev) => {
```